mlir/include/mlir/Dialect/OpenMP/OpenMPClauseOperands.h

```diff
@@ -179,46 +179,46 @@ struct PrivateClauseOps {
   llvm::SmallVector<Value> privateVars;
   // The list of symbols referring to delayed privatizer ops (i.e. `omp.private`
   // ops).
-  llvm::SmallVector<Attribute> privatizers;
+  llvm::SmallVector<Attribute> privateSyms;
 };
 
 struct ProcBindClauseOps {
-  ClauseProcBindKindAttr procBindKindAttr;
+  ClauseProcBindKindAttr procBindKind;
 };
 
 struct ReductionClauseOps {
   llvm::SmallVector<Value> reductionVars;
```

Comment:
```suggestion
  llvm::SmallVector<Value> reduction;
```
For other clauses, the main clause argument just has the clause's name as well (e.g. `OrderClauseOps`). Why different for reduction, task_reduction, ...?

Reply:
The reason some clauses like `OrderClauseOps` have an argument with just the clause name is not only that it's the "main" argument, but also that it's a single value or attribute. However, this argument falls into the variadic arguments case, to which the "vars" suffix is added.

Maybe evaluating this case in isolation, the choice between `reductions` or `reductionVars` could be argued either way. But there are other cases where not adding the prefix makes it read weird, so for consistency I proposed adding it to all of them. For example: `linearVars` vs `linears`, `isDevicePtrVars` vs `isDevicePtrs` or `privateVars` vs `privates` (this one is particularly bad).